y/index.md

```diff
@@ -0,0 +1,5 @@
+List of artists starting with Y
+
+- [Yoann Loic](yoann-loic.md)
+
+Go back to the artist index
```

Comment:
This text needs to be a link, and the link should go back to the root index.md file that Sugi and Clopedia are adding.